common/config/config.json.template

```diff
@@ -5,6 +5,7 @@
   "rootFolderPath" : "~/kLive_content/live",
   "oldContentFolderPath" : "~/kLive_content/archive",
   "requestTimeout" : 20000,
+  "flavorSourceId" : "32",
```

Comment:
not good, never estimate that

Reply:
there's not way for me to know which flavor is ingested (source or not) without making another API call
and I don't wanna do it